scripts/add_electricity.py

```diff
@@ -247,6 +266,14 @@ def attach_load(n, regions, load, admin_shapes, countries, scale):
         gpd.read_file(regions).set_index("name").reindex(substation_lv_i)
     ).dropna(
         axis="rows")  # TODO: check if dropna required here. NaN shapes exist?
+
+    weather_year = snakemake.config["load_options"]["weather_year"]
+    prediction_year = snakemake.config["load_options"]["prediction_year"]
+    region_load = snakemake.config["region_load"]["scale"]
```

Comment:
Another note. Be careful with the assignment. I think you mean here:
region_load = snakemake.config["load_options"]["regions_load"]    ?
(Again this assignment needs to be placed before the function is executed and not in the function.)